lfs/ssh_test.go

```diff
@@ -0,0 +1,442 @@
+package lfs
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/bmizerany/assert"
```

Comment:
Change this to "github.com/github/git-lfs/vendor/_nuts/github.com/technoweenie/assert" to get the test to pass.